packages/babel-parser/test/fixtures/flow/expect-plugin/export-interface/options.json

```diff
@@ -1,7 +1,8 @@
 {
   "sourceType": "module",
   "plugins": [
-    "jsx"
+    "jsx",
+    "estree"
```

Comment:
We don't need to add `"estree"` plugin for flow tests -- All parser tests will be run against `estree` plugin by `estree-throws.js`

Reply:
This was automatically added by `OVERWRITE=true make test`; I didn’t notice this one and should have deleted it. I will do so soon.